### DISCUSSION TITLE
Compatibility Issue with the FooEvents For Woocommerce. Attendee fields does not show in the checkout page.

### DISCUSSION THREAD

**Discussion by Juha83** [woocommerce/woocommerce#45074] [WooCommerce, Blocks, and Site Editing]
## Describe the bug
I am using a FooEvents for Woocommerce plugin and the plugin should display fillable attendee fields on the checkout page.
But for some reason, Woocommerce Blocks is blocking it.

## To reproduce

1. Go to https://playground.juhakumpula.com/product/liverpool/
3. Add the product into a cart
3. Go to https://playground.juhakumpula.com/checkout/
4. Click Proceed to checkout
5. There should be the fields Product name, variation name, and fillable fields First name, Last name, and email address but it does not show.

## Expected behavior

Attendee fields do not show on the checkout page.

### WordPress (please complete the following information):
* Core version: 5.7.2
* Gutenberg version: Not installed
* WooCommerce version: 5.4.1
* WooCommerce Blocks version: 5.5.0
* Site language: English
* Any other plugins installed:

**NOTE**

I tried to turn off every plugin that I had except Woocommerce, Woocommerce Blocks, and the FooEvents for Woocommerce.
It only works when the default [woocommerce_checkout] is set.

Contact Form 7 | Version 5.4.1
Duplicate Page | Version 4.4
Elementor | Version 3.2.5
Elementor Pro | Version 3.3.1
Email Template Customizer for WooCommerce | Version 1.0.4.1
Facebook for WooCommerce | Version 2.6.1
FooEvents Custom Attendee Fields | Version 1.5.12
FooEvents Express Check-in | Version 1.5.7
FooEvents for WooCommerce | Version 1.12.36
FooEvents PDF Tickets | Version 1.8.12
GP Premium | Version 2.0.2
Insert Headers and Footers | Version 1.6.0
OMGF | Version 4.4.3
Ultimate FAQs | Version 2.0.19
Variation Swatches for WooCommerce | Version 1.1.17
Variation Swatches for WooCommerce - Pro | Version 1.1.17
Visibility Logic for Elementor | Version 2.1.5
WooCommerce | Version 5.4.1
WooCommerce Blocks | Version 5.5.0
WooCommerce Extra Fees Plugin | Version 3.6.4
WooCommerce Product Tabs | Version 2.0.3
WooLentor - WooCommerce Elementor Addons + Builder | Version 1.9.0
WooLentor Pro | Version 1.6.7

### Desktop (please complete the following information):
* OS: Windows 10 Pro, Version 21H1, OS build 19043.1055
* Browser: Chrome
* Version [e.g. 22]

### Smartphone (please complete the following information):
* Device: [e.g. iPhone6]
* OS: [e.g. iOS8.1]
* Browser [e.g. stock browser, safari]
* Version 91.0.4472.124 (Official Build) (64-bit)

## Additional context
See the screenshots

![Attendee Fields](https://user-images.githubusercontent.com/62006154/125039494-3c3d3c00-e09f-11eb-85e5-9ecfdd688ea8.JPG)
![Error](https://user-images.githubusercontent.com/62006154/125039500-3cd5d280-e09f-11eb-8cca-e6947791556c.JPG)

**Comment by mikejolley** [comment]
Hi there @Juha83 

The new block-based checkout does not work in the same way as the old checkout and requires a dedicated integration, which FooEvents does not yet appear to have. As such, you won't be able to use FooEvents and the new Checkout together, yet.

In the coming weeks, we'll be expanding what integration points are available on the block-based checkout and documenting these for developers. At which point it will be possible for FooEvents team to support Checkout if they wish. In the meantime, you may want to simply mention this as a request to FooEvents so it's on their radars.

Thanks
